docker/Dockerfile

```diff
@@ -272,13 +262,24 @@ RUN install-package \
 
 # Install formatters as last step so if they change the image is easy to update
 RUN pip install --upgrade \
-    cmake-format==0.6.9 \
+    cmake-format==0.6.10 \
     isort==4.3.21 \
     black==19.10b0
 
-#######################################
-### ADD NEW PROGRAMS/LIBRARIES HERE ###
-#######################################
+##############################
+### SETUP THE USER ACCOUNT ###
+##############################
+
+# Setup the sudo so it can be run without a password
+COPY etc/sudoers.d/user /etc/sudoers.d/user
+RUN chmod 440 /etc/sudoers.d/user
+
+# Create the fourtel user, and setup sudo so no password is required
```

Comment:
Excuse me? Which user??

Reply:
This is why you don't put constants in comments :P